### DISCUSSION TITLE
Live Stitching

### DISCUSSION THREAD

**Discussion by davidhardeniv-code** [OpenStitching/stitching#221] [Ideas]
I was wondering is there any capabilities to do a live image stitching like with a webcam attached or rtsp feed live. I was just wondering, thanks!

**Comment by lukasalexanderweber** [comment]
This should be possible. You can stitch numpy arrays since [v0.5.0](https://github.com/OpenStitching/stitching/releases/tag/v0.5.0) and there is also some progress in [Video Stitching](https://github.com/OpenStitching/stitching/discussions/148)
